src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+p,
+h1,
+h2 {
+  margin: 0;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
```

Comment:
font-weight: bold is a default font-weight for h1 headings, let's omit it in styles